app/controllers/api/v1/applications_controller.rb

```diff
@@ -4,32 +4,32 @@ def index
   end
 
   def show
-    if build_application.present?
```

Comment:
Hmm why the change from `build_`?

Reply:
Oops, this was my mistake when resolving a merge - intended to stick with `build_`! Will fix it.